dockerfile/README.md

```diff
@@ -1,34 +1,33 @@
-# Dockerfile for WSO2 Identity Server #
+# Dockerfile for WSO2 Identity Server
 The Dockerfile defines the resources and instructions to build the Docker image for WSO2 Identity Server 5.3.0.
 
 ## Build and run
```

Comment:
Shall we update above using the same pattern as of 'How to build an image and run' in similar to 'How to update conf' ?